### PR TITLE
Issue/3030 shipping labels banner

### LINKED ISSUE #3030: Shipping Labels M1: add feature announcement banner in Order Detail
As part of the Shipping Labels M1 release, we need to add the following dismissible feature banner in the Order detail screen.

**This will only be displayed in there are shipping labels available for the order.**

### PR DESCRIPTION
**Pull request by anitaa1990**
Fixes #3030 by adding a feature announcement banner for shipping labels, if WooCommerce Shipping is available for the store.

#### Changes
- Modified `ProductsWIPNoticeCard` to `FeatureWIPNoticeCard` to make it reusable for all new features in the app.
- Added new feedback url for asking feedback from users for shipping labels.
- Added new banner to Order detail to announce the Shipping Labels feature.

#### Screenshots
. . . 

#### Testing
- Click on an order with Shipping Labels. Notice the announcement banner at the top of the screen.
  - Click on the `Give Feedback` button and notice the survey webview. Add your feedback and notice that the your feedback is given and you are redirected to the feedback confirmation screen.
  - Click on the `Dismiss` button and notice that the banner is dismissed and no longer visible.
- Click on an order **without** Shipping Labels. Notice the announcement banner is not displayed.
- Click on the Products tab. Notice the feature announcement banner at the top of the screen. 
  - Click on the `Give Feedback` button and notice the survey webview. Add your feedback and notice that the your feedback is given and you are redirected to the feedback confirmation screen.
  - Click on the `Dismiss` button and notice that the banner is dismissed and no longer visible.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/OrderDetailFragment.kt [hunk 1]
```diff
@@ -285,7 +305,50 @@ class OrderDetailFragment : BaseFragment(), NavigationResult, OrderProductAction
                     }
                 )
             }
-        }.otherwise { orderDetail_shippingLabelList.hide() }
+        }.otherwise {
```
Comment (0nko) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/OrderDetailFragment.kt:308]: This is cool!

**Review by 0nko — COMMENTED** [review]
The banner works well! There's just one small nit-pick -- when you tap on the banner, the background selection is not bounded by the banner frame like you'd see with a button or a card.

**Comment by anitaa1990** [comment]
Thanks for the review @0nko! 

> When you tap on the banner, the background selection is not bounded by the banner frame like you'd see with a button or a card. 

Imho, this looks to be a problem in `develop` as well. I think it might have something to do with the fact that we use `?android:attr/selectableItemBackgroundBorderless` for the `Woo.Card.ExpanderButton` background. Would you mind if we open a separate issue for that since it would affect all the expandable cards used in the app.

**Comment by 0nko** [comment]
Sure!

**Review by 0nko — APPROVED** [review]
:shipit:

**Comment by anitaa1990** [comment]
Thank you @0nko 🙇‍♀️

**Pull request merged**